docs/APISocketOptions.md

```diff
@@ -0,0 +1,1511 @@
+
+- When this flag is set to 1 on a listener socket, it allows this socket to
+accept group connections. When set to the default 0, group connections will be
+rejected. Keep in mind that if the `SRTO_GROUPCONNECT` flag is set to 1 (i.e.
+group connections are allowed) `srt_accept` may return a socket **or** a group
+ID. A call to `srt_accept` on a listener socket that has group connections
+allowed must take this into consideration. It's up to the caller of this
+function to make this distinction and to take appropriate action depending on
+the type of entity returned.
+
+- When this flag is set to 1 on an accepted socket that is passed to the
+listener callback handler, it means that this socket is created for a group
+connection and it will become a member of a group. Note that in this case
+only the first connection within the group will result in reporting from
+`srt_accept` (further connections are handled in the background), and this
+function will return the group, not this socket ID.
+
+
+[Return to list](#list-of-options)
+
+
+
+#### SRTO_GROUPSTABTIMEO
+
+| OptName               | Since | Binding | Type       | Units  | Default  | Range  | Dir | Entity |
+| --------------------- | ----- | ------- | ---------- | ------ | -------- | ------ | --- | ------ |
+| `SRTO_GROUPSTABTIMEO` |       | pre     | `int32_t`  | ms     | 40       | 10-... | W   | GSD+   |
+
+- This setting is used for groups of type `SRT_GTYPE_BACKUP`. It defines the stability 
+timeout, which is the maximum interval between two consecutive packets retrieved from 
+the peer on the currently active link. These two packets can be of any type,
+but this setting usually refers to control packets while the agent is a sender.
+Idle links exchange only keepalive messages once per second, so they do not
+count. Note that this option is meaningless on sockets that are not members of
+the Backup-type group.
+
+- This value should be set with a thoroughly selected balance and correspond to
+the maximum stretched response time between two consecutive ACK messages. By default
+ACK messages are sent every 10ms (so this interval is not dependent on the network
+latency), and so should be the interval between two consecutive received ACK
+messages. Note, however, that the network jitter on the public internet causes
+these intervals to be stretched, even to multiples of that interval. Both large
+and small values of this option have consequences:
+
+- Large values of this option prevent overreaction on highly stretched response
+times, but introduce a latency penalty - the latency must be greater
+than this value (otherwise switching to another link won't preserve
+smooth signal sending). Large values will also contribute to higher packet
+bursts sent at the moment when an idle link is activated.
+
+- Smaller values of this option respect low latency requirements very
+well, may cause overreaction on even slightly stretched response times. This is
```

Comment:
As the second part of this sentence is a drawback warning and contradicts the first part of the sentence being an advantage announcement, I think the second part definitely deserves some "however" or "unfortunately".